.gitignore

```diff
@@ -15,6 +15,12 @@ pids
 *.seed
 *.pid.lock
 
+# SQLite board server local db
+*.db
+
+# output from markdown task
+all_markdown.md
```

Comment:
This was something I missed out from the previous PR that introduced that task, so I thought since I was making changes to the gitignore anyway I would add this at the same time.